testplan/vendor/tempita/__init__.py

```diff
@@ -30,16 +30,17 @@ def foo(bar):
 
 import re
 import sys
+import os
+import tokenize
+
 try:
-    from urllib.parse import quote as url_quote
     from io import StringIO
     from html import escape as html_escape
```

Comment:
I think `cgi.escape` exists in both python 2 and 3. Any particular reason to use `html.escape` instead for python3?